server/market/market.go

```diff
@@ -2137,7 +2141,7 @@ func (m *Market) prepEpoch(orders []order.Order, epochEnd time.Time) (cSum []byt
 		// Change the order status from orderStatusEpoch to orderStatusRevoked.
 		coid, revTime, err := m.storage.RevokeOrder(ord)
 		if err == nil {
-			m.auth.RecordCancel(ord.User(), coid, ord.ID(), revTime)
+			m.auth.RecordCancel(ord.User(), coid, ord.ID(), db.EpochGapNA, revTime)
```

Comment:
I'm thinking this one should perhaps use a gap of 0.  Yes, the user will get the preimage miss strike too, but this is effectively cancelling an order in the same epoch.  Redundant with the preimage violation?